src/main/webapp/resources/js/layouts/project-layout.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Outlet, useParams } from "react-router-dom";
+import { PageHeader } from "antd";
+import { useGetProjectDetailsQuery } from "../redux/endpoints/project";
+import ProjectNavigation from "../components/project/project-navigation";
+
+/**
+ * React component for the layout of the project specific pages
+ * @constructor
+ */
+export default function ProjectLayout(): JSX.Element {
+  const { projectId } = useParams();
+
+  const { data: details = {} } = useGetProjectDetailsQuery(projectId);
+
+  return (
+    <PageHeader
+      title={details.label}
+      subTitle={details.description}
+      style={{ margin: `0 25px` }}
+    >
+      <div style={{ backgroundColor: `#ffffff` }}>
```

Comment:
Get variable from app.css for the background color

Reply:
Can't, app.css is not loaded onto this page. Still thinking about how to handle this.

Reply:
Sounds good. We can handle that in a later pr